src/Okta.Idx.Sdk/SocialLoginSettings.cs

```diff
@@ -0,0 +1,30 @@
+﻿// <copyright file="IdxState.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using Newtonsoft.Json;
+using Okta.Idx.Sdk.Configuration;
+
+namespace Okta.Idx.Sdk
+{
+    /// <summary>
+    /// A descriptor for the current state of an Idx social login interaction.
+    /// </summary>
+    public class SocialLoginSettings
```

Comment:
Should we rename this to `SocialLoginResponse` to be consistent with other response objects?

Reply:
This object is intended to represent the settings for social-login and doesn't necessarily represent any response from the api.  It's intended to be used at the application layer where the response/request analogy is more useful in reference to classes related to the Http protocol specifically.

I suppose I could be convinced that this is a response to the component that renders the html, maybe "options" would be more appropriate?  Open to more thoughts here.